development/scripts/clone-repos.sh

```diff
@@ -21,7 +20,5 @@ do
     cd "${repo}"/
     git remote add upstream git@github.com:opensecuritycontroller/"${repo}".git
     git remote set-url --push upstream no_push
-    cd ../"${repo}"/.git/hooks 
-    ln -s ../../hooks/pre-push .
-    cd ../../../
+    cd ../

```

Comment:
I think we should keep the hooks, they dont add maintenance since they should never need update and they add an extra safe net in case someone clones and sets the remote differently or misses a step.